tests/test_labeling.py

```diff
@@ -0,0 +1,154 @@
+# Copyright (C) 2019-2022 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+import os.path as osp
+from unittest.case import TestCase
+
+import numpy as np
+
+from datumaro.components.annotation import AnnotationType, Label, LabelCategories
+from datumaro.components.extractor import DatasetItem
+from datumaro.components.media import Image
+from datumaro.components.project import Dataset
+
+from .requirements import Requirements, mark_requirement
+
+TMP_DATASET_DIR = osp.join(osp.dirname(__file__), "tmp", "test_labeling")
+
+
+class LabelingTest(TestCase):
+    @mark_requirement(Requirements.DATUM_GENERAL_REQ)
+    def test_multiple_label(self):
+        label_categories = LabelCategories()
+        label_categories.add("car", parent="")
+        label_categories.add("bicycle", parent="")
+
+        dataset = Dataset.from_iterable(
+            [
+                DatasetItem(
+                    id=0,
+                    subset="train",
+                    media=Image(data=np.ones((10, 6, 3))),
+                    annotations=[
+                        Label(
+                            0,
+                            id=0,
+                        ),
+                        Label(
+                            1,
+                            id=1,
+                        ),
+                    ],
+                ),
+            ],
+            categories={
+                AnnotationType.label: label_categories,
+            },
+        )
```

Comment:
You can initialize shared resources across tests in `setUpClass`. Please refer to https://docs.python.org/3/library/unittest.html#class-and-module-fixtures.

Reply:
Every test case is using slightly different LabelCategories or DatasetItems. Thus I think that we don't need to use shared resources in this time.